lib/spack/spack/test/cmd/python.py

```diff
@@ -3,12 +3,34 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
+import pytest
+
 import spack
 from spack.main import SpackCommand
 
 python = SpackCommand('python')
 
+test_string = "Ran the spack python module"
```

Comment:
Wondering if there's a test that was half-coded here, or if I am missing something...

Reply:
yeah that's exactly what happened -- thanks!